### PR TITLE
Spring boot day2

### PR DESCRIPTION
**Pull request by renukaThink**
Insert,Delete and select Queries in JDBC
tried JDBC  sql commands in dynamic way

### CODE REVIEW

File: day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/day2/JPARepository.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import jakarta.persistence.EntityManagerFactory;
+import jakarta.persistence.PersistenceContext;
+import jakarta.transaction.Transactional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Repository;
+
+@Repository
+@Transactional
+public class JPARepository {
```
Comment (yogesh-think) [inline day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/day2/JPARepository.java:14]: Use Pascal case to define class name

File: day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/day2/JDBCRepository.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package com.learnspringjpahibernate.springjpahibernate.day2;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.BeanPropertyRowMapper;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Repository;
+@Component
+@Repository
+public class JDBCRepository {
```
Comment (yogesh-think) [inline day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/day2/JDBCRepository.java:10]: Use Pascal case to define class name

File: day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/day2/Employee.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    }
+
+    public Employee(int id, String name, String designation, int salary) {
+        this.id = id;
+        this.name = name;
+        this.designation = designation;
+        this.salary = salary;
+    }
+
+    public Employee() {
```
Comment (yogesh-think) [inline day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/day2/Employee.java:55]: Don't need to write non parameterized constructor

File: day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/Student.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package com.learnspringjpahibernate.springjpahibernate;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.stereotype.Component;
+
+@Component
```
Comment (yogesh-think) [inline day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/Student.java:6]: Is it @Entity or @Component?

File: day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/SpringJpaHibernateApplication.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+import com.learnspringjpahibernate.springjpahibernate.day2.Employee;
+import com.learnspringjpahibernate.springjpahibernate.day2.JDBCRepository;
+import com.learnspringjpahibernate.springjpahibernate.day2.JPARepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.ImportResource;
+
+@ComponentScan("package com.day2")
```
Comment (yogesh-think) [inline day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/SpringJpaHibernateApplication.java:12]: Check defined package path

File: day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/day2/JDBCRepository.java (hunk 2 above)
Comment (renukaThink) [reply day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/day2/JDBCRepository.java:10]: okay

File: day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/day2/Employee.java (hunk 3 above)
Comment (renukaThink) [reply day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/day2/Employee.java:55]: okay

File: day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/Student.java (hunk 4 above)
Comment (renukaThink) [reply day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/Student.java:6]: it's @Entity, I will correct it.

File: day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/SpringJpaHibernateApplication.java (hunk 5 above)
Comment (renukaThink) [reply day1-spring/springjpahibernate/spring-jpa-hibernate/src/main/java/com/learnspringjpahibernate/springjpahibernate/SpringJpaHibernateApplication.java:12]: okay

**Pull request merged**